beets/ui/commands.py

```diff
@@ -1666,6 +1666,7 @@ def print_completion(*args):
     u'/usr/share/local/bash-completion/bash_completion',
```

Comment:
I know you didn't change this @bismark, but this looks like it may be a spelling mistake as I don't know of any systems that have a `/usr/share/local` (should be `/usr/local/share`).